spec/PasswordPolicy.spec.js

```diff
@@ -124,41 +127,53 @@ describe("Password Policy: ", () => {
         resetTokenValidityDuration: 0
       },
       publicServerURL: "http://localhost:8378/1"
-    })
-      .then(() => {
-        fail('resetTokenValidityDuration negative number test failed');
-        done();
-      })
-      .catch(err => {
-        expect(err).toEqual('passwordPolicy.resetTokenValidityDuration must be a positive number');
-        done();
-      });
+    }).then(() => {
+      fail('resetTokenValidityDuration negative number test failed');
+      done();
+    }).catch(err => {
+      expect(err).toEqual('passwordPolicy.resetTokenValidityDuration must be a positive number');
+      done();
+    });
   });
 
-  it('should fail if passwordPolicy.validator setting is invalid type', done => {
+  it('should fail if passwordPolicy.validatorPattern setting is invalid type', done => {
     reconfigureServer({
       appName: 'passwordPolicy',
       passwordPolicy: {
-        validator: 1 // number is not a valid setting for validator
+        validatorPattern: "abc" // string is not a valid setting
       },
       publicServerURL: "http://localhost:8378/1"
-    })
-      .then(() => {
-        fail('passwordPolicy.validator type test failed');
-        done();
-      })
-      .catch(err => {
-        expect(err).toEqual('passwordPolicy.validator must be a RegExp, a string or a function.');
-        done();
-      });
+    }).then(() => {
+      fail('passwordPolicy.validatorPattern type test failed');
+      done();
+    }).catch(err => {
+      expect(err).toEqual('passwordPolicy.validatorPattern must be a RegExp.');
+      done();
+    });
   });
 
-  it('signup should fail if password does not confirm to the policy enforced using RegExp', (done) => {
+  it('should fail if passwordPolicy.validatorCallback setting is invalid type', done => {
+    reconfigureServer({
+      appName: 'passwordPolicy',
+      passwordPolicy: {
+        validatorCallback: "abc" // string is not a valid setting
+      },
+      publicServerURL: "http://localhost:8378/1"
+    }).then(() => {
+      fail('passwordPolicy.validatorCallback type test failed');
+      done();
+    }).catch(err => {
+      expect(err).toEqual('passwordPolicy.validatorCallback must be a function.');
+      done();
+    });
+  });
+
+  it('signup should fail if password does not confirm to the policy enforced using validatorPattern', (done) => {
```

Comment:
@bhaskaryasa 
A few more minor nits

pls change `confirm` to `conform` across all the test cases